### ISSUE TITLE
What kind of crappy project is this? It's too useless

### ISSUE THREAD

**Issue by haisheng666** [TemryL/ComfyUI-IDM-VTON#66]
![image](https://github.com/user-attachments/assets/91c37f5c-34ae-46c3-944f-9a21daf10f8d)
What kind of crappy project is this? It's too useless

**Comment by TemryL** [comment]
Hi, thanks for your kind feedback. The issue comes from the mask generation with SAM. Your mask should not cover the full body, only the part where the garment should be worn. Maybe try something like 'sweater' instead of 'hoodie' or find another way to generate your mask. Let me know if that’s helpful.

**Comment by haisheng666** [comment]
Okay, I'll come down and study it carefully again
---- Replied Message ----
From Tom ***@***.***> Date 07/24/2024 20:43 To ***@***.***> Cc ***@***.***>***@***.***> Subject Re: [TemryL/ComfyUI-IDM-VTON] What kind of crappy project is this? It's too useless (Issue #66) 
Hi, thanks for your kind feedback. The issue comes from the mask generation with SAM. Your mask should not cover the full body, only the part where the garment should be worn. Maybe try something like 'sweater' instead of 'hoodie' or find another way to generate your mask. Let me know if that’s helpful.

**Comment by Danny-zhang-xu** [comment]
Exception during processing!!! unsupported operand type(s) for *: 'NoneType' and 'int'

Have you ever encountered the above situation? How to solve it?thanks!

**Comment by Pythonpa** [comment]
> ![image](https://private-user-images.githubusercontent.com/42937758/351671927-91c37f5c-34ae-46c3-944f-9a21daf10f8d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM0NDI0MzMsIm5iZiI6MTcyMzQ0MjEzMywicGF0aCI6Ii80MjkzNzc1OC8zNTE2NzE5MjctOTFjMzdmNWMtMzRhZS00NmMzLTk0NGYtOWEyMWRhZjEwZjhkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEyVDA1NTUzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThjM2UxZDVkYWFlMWMxMTFmMjIzNGJlYTNkYzAwMzNlNzdmZjc2MjYxMzdmZjc1NzI4ZTlhNzMyOGYwYTcyYzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.H3fQFmjt9Ohw9gJTSBdLREULIhX0N5JuJc0Ae8Uznjc) What kind of crappy project is this? It's too useless

If you don't know how to use it yourself, please learn to use some basic methods before using this project. In addition, I think we should pay tribute to the open source spirit of the author, instead of just publishing arrogant and rude complaints

**Comment by cristian-tapia** [comment]
> If you don't know how to use it yourself, please learn to use some basic methods before using this project. In addition, I think we should pay tribute to the open source spirit of the author, instead of just publishing arrogant and rude complaints.

I completely agree. There's no reason to be rude or disrespectful. I've had positive results with it, and even if I hadn't, I wouldn't feel right posting such an unconstructive comment here.
